src/invocation/ClusterService.ts

```diff
@@ -173,6 +174,17 @@ class ClusterService extends EventEmitter {
         return this.ownerConnection;
     }
 
+    /**
+     * Returns random connection
+     * @returns {Address}
+     */
+    getRandomAddress(): Address {
```

Comment:
I think this method is out of this pr's scope. See my comment about `encodeInvokeOnRandomPartition` method to see why.